scripts/themer

```diff
@@ -84,7 +84,7 @@ def get_filters():
         return get_component(color, 2)
 
     def to_float(num):
-        return num / 255
+        return num / 255.
```

Comment:
Yeah - the `num` parameter is an `int` when passed to the function. The `/` operation in Python will do `div` rather than divide when using `int`s (so `5 / 2` is `2`). To force float division you you can either do `float(5) / 2` or `5 / 2.` - making any number a `float` forces the entire calculation (and result) to be done as `float` arithmetic.